src/data/fintype/basic.lean

```diff
@@ -1517,6 +1544,11 @@ assume (hf : surjective f),
 have H : infinite α := infinite.of_surjective f hf,
 by exactI not_fintype α
 
+instance function.embedding.fintype' {α β} [infinite α] [fintype β] : fintype (α ↪ β) :=
```

Comment:
```suggestion
-- the instance generated by `is_empty → subsingleton → fintype is non-computable
instance function.embedding.fintype' {α β} [infinite α] [fintype β] : fintype (α ↪ β) :=
```

Reply:
I think probably we should add an `is_empty.fintype` instance to handle this problem elsewhere; but not need to do so in this PR.

Reply:
yeah, was going to do it when it gets merged to master